bin/build-image

```diff
@@ -3,25 +3,25 @@
 set -o errexit
 
 GIT_ROOT="${GIT_ROOT:-$(git rev-parse --show-toplevel)}"
-. "${GIT_ROOT}/bin/include/versioning"
```

Comment:
The reason for changes is by some script check tool?

Reply:
No, just removing useless expansion brackets. They're not recommended by shellcheck and I think they give a false sense of security and add line noise. The `${GIT_ROOT}` is not using variable expansion features like in the line above.

This line from the PR for example:
`DOCKER_IMAGE_TAG=${VERSION_TAG}` is identical to `DOCKER_IMAGE_TAG=$VERSION_TAG` and both are actually protected from pathname expansion and word splitting. However I still would add double quotes to be consistent: `DOCKER_IMAGE_TAG="$VERSION_TAG"`.